docs/internals/contributing/writing-code/pull-requests.txt

```diff
@@ -98,6 +98,8 @@ hoops of claiming tickets. Submit your changes directly and you're done!
 It is *always* acceptable, regardless whether someone has claimed it or not, to
 link proposals to a ticket if you happen to have the changes ready.
 
+.. TODO: link
+
```

Comment:
```suggestion
```

Reply:
🤦🏻  oops, that shouldn't have been there